apps/src/assetManagement/assetPrefix.js

```diff
@@ -82,9 +82,11 @@ export function fixPath(filename) {
   if (SOUND_PREFIX_REGEX.test(filename)) {
     return filename.replace(SOUND_PREFIX, soundPathPrefix);
   }
-
+  const state = getStore().getState();
```

Comment:
A comment explaining this case might be helpful

Reply:
Good idea! Just added - thanks!